plugins/module_utils/_api_data.py

```diff
@@ -44,7 +44,7 @@ def __init__(self,
             # Mark as 'fully understood' if it is for at least one version
             self.fully_understood = False
             for dummy, dummy, unversioned in self.versioned:
-                if unversioned.fully_understood:
+                if not isinstance(unversioned, str) and unversioned.fully_understood:
```

Comment:
It can also be `None`.
```suggestion
                if unversioned and not isinstance(unversioned, str) and unversioned.fully_understood:
```